.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
+package-lock.json
```

Comment:
Ignoring the package-lock file to prevent merge conflicts in future pull requests.

Reply:
@cmneal26 

I don't really understand much about this git stuff. As a junior technical writer, using a docs-as-code approach, how often do you think I'll have to resolve git conflicts, using a similar GitHub Collaborative strategy? Is this something, as a technical writer that is more technical, I should focus on, or is this more a "Software engineers love it, but isn't as much of a priority for a technical writer?"

I guess, as a junior technical writer, what I'm trying to say is this, "I sort of understand this docs as code approach now, because we did some git stuff. But this conflict resolution of branches seems hard to me. Is this something I should worry or focus on as a junior technical writer aiming for a docs-as-code approach, or is this something relevant, but maybe something to worry about for a more senior technical writer role, since although we are technical writers, we aren't software engineers?

Tim, help me! I don't know and I want your perspective!

Reply:
I doubt technical writers need to deal with merge conflicts for **code** changes often. I suspect conflicts in code would only occur during the initial setup of the structure. Once the code has stabilized, any future conflicts would be in **content** changes. Content changes ought to be more manageable by technical writers of all levels. 

Although conflicts would be rare if each writer is working on different content files, conflicts would still occur when shared content, such as glossary of terms, is changed. Merging in git is not too complex once one has a chance to practice a bit. I suspect having a working knowledge of git commands (`add`, `commit`, `pull`, `push`, `merge`, `reset`) is required for any tech writers treating documentation as code. 

`git` is a tool worth investing some quality learning time.
Thankfully, there are many free git tutorials available.